File: java_project_front/lib/List/produkt.dart
class Produkt {
  final int id;
  final String name;
  final double quantity;
  final String unit;

  Produkt({required this.id, required this.name, required this.quantity, required this.unit});

  factory Produkt.fromJson(Map<String, dynamic> json) {
    return Produkt(
      id: json['id'],
      name: json['name'],
      quantity: json['quantity'],
      unit: json['unit'],
    );
  }
}

File: demo/src/main/java/com/example/demo/produkt.java
package com.example.demo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class produkt {
    public int id;
    public String name;
    public double quantity;
    public int unit;
}

